src/gitgraph.js

```diff
@@ -1505,9 +1557,102 @@
     return (typeof object === "object");
   }
 
+  /**
+   * Returns `true` if any of the properties (nested or single) of `obj` specified by `key` are undefined or set to a value of null.
+   * Modified from original source: http://stackoverflow.com/a/23809123.
+   *
+   * @param {*} obj - The object whose properties are to be tested as being undefined or equal to null.
+   * @param {String} key - The property hierarchy of `obj` to be tested, specified using 'dot notation' (e.g. property1.property2.property3 etc).
+   * @returns {Boolean} - True if ANY of the properties specified by `key` is undefined or equal to null, otherwise False.
+   * @private
+   */
+  function _isNullOrUndefined(obj, key) {
+
+    /* We invert the result of '.every()' in order to meet the expected return value for the condition test of the function.
+     * We have to do this, given that '.every()' will return immediately upon capturing a falsey value from the callback.
+     *
+     * See: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/every for more information.
+     */
+    return !(key.split(".").every(function(x) {
+        if(typeof obj !== "object" || obj === null || !(x in obj)) {
+          return false;
+        }
+        obj = obj[x];
+        return true;
+    }));
+  }
+
+  /* Polyfill for ECMA-252 5th edition Array.prototype.every()
+   * See: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/every
+   * for more information.
+   * */
+  if (!Array.prototype.every) {
+    Array.prototype.every = function(callbackfn, thisArg) {
+      var T, k;
+
+      if (this === null) {
+        throw new TypeError("this is null or not defined");
+      }
+
+      // 1. Let O be the result of calling ToObject passing the this
```

Comment:
Thanks for the detailed polyfill and the link to the source, that's helpful (and documented).

Still, you can get rid of all the detailed implementation comments as they mostly describe what the code is doing rather than why, which is not really useful.

Thanks a lot =)